code/datums/station_traits/negative_traits.dm

```diff
@@ -134,3 +134,86 @@
 		/// The bot's language holder - so we can randomize and change their language
 		var/datum/language_holder/bot_languages = found_bot.get_language_holder()
 		bot_languages.selected_language = bot_languages.get_random_spoken_language()
+
+/datum/station_trait/revenge_of_pun_pun
+	name = "Revenge of Pun Pun"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	var/static/list/weapon_types
```

Comment:
why is this static?

Reply:
To reiterate what was said in discord: because I was making no assumption as to the amount of instances of this that get made, even if this is a singleton it indicates to people reading it that it's a value that only needs to be generated once.